configs/base.eslintrc.json

```diff
@@ -7,37 +8,28 @@
       "jsx": true
     }
   },
-  "plugins": [
-    "@typescript-eslint",
-    "header",
-    "import",
-    "no-null",
-    "react"
-  ],
+  "plugins": ["@typescript-eslint", "header", "import", "no-null", "prettier"],
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:import/errors",
     "plugin:import/warnings",
     "plugin:import/typescript",
-    "plugin:react/recommended"
+    "prettier"
   ],
-  "settings": {
-    "react": {
-      "createClass": "createReactClass",
-      "pragma": "React",
-      "version": "detect",
-      "flowVersion": "0.53"
-    }
-  },
   "env": {
     "browser": true,
     "mocha": true,
-    "es2017": true
+    "es6": true
```

Comment:
why use es6 here when [configs/base.tsconfig.json](https://github.com/eclipse-emfcloud/theia-tree-editor/pull/73/files#diff-69540381ae31eb557dbe0862c952999d8efe4e4e3b5bc84cc1916f8e440686d6) uses es2017 as lib and target

Reply:
I copied it over from the `emfcloud-modelserver-theia` config . This is the feature set for eslint, there is no option to use es2017 features. es6 is the only offered option in that regard.